parsl/config.py

```diff
@@ -50,6 +50,9 @@ class Config(RepresentationMixin):
         of 1.
     run_dir : str, optional
         Path to run directory. Default is 'runinfo'.
+    std_autopath : function, optional
+        Sets the function used to generate stdout/stderr specifications when parsl.AUTO_LOGPATH is used. If no function
+        is specified, generates paths that look like: ``rundir/NNN/task_logs/X/task_{id}_{name}{label}.{out/err}``
```

Comment:
`NNN` is clear to me; what is `X` in the path name?

Reply:
it's ` str(int(task_id / 10000)).zfill(4)` - the 10000s part of the task ID and upwards. I omitted a human description because it's awkwardly verbose for what I intended to be a shortish docstring... I guess something based around "dividing into directories with 10000 tasks each" would work?